radio/src/gui/480x272/lcd.h

```diff
@@ -133,7 +133,7 @@ void lcdDrawNumber(coord_t x, coord_t y, int32_t val, LcdFlags flags=0, uint8_t
 
 void putsModelName(coord_t x, coord_t y, char *name, uint8_t id, LcdFlags att);
 void putsStickName(coord_t x, coord_t y, uint8_t idx, LcdFlags att=0);
-void drawSwitch(coord_t x, coord_t y, swsrc_t swtch, LcdFlags flags=0);
+void drawSwitch(coord_t x, coord_t y, int32_t swtch, LcdFlags flags=0);
```

Comment:
The compiler can't find the definition of swsrc_t when building this way, presumably due to some bizarre include order mixup in the amalgamated source file (? even if openth.h is included at the top of lcd.h... I don't really get it either, but it's hard to follow what depends on what due to the reliance on amalgamation in the end.  My IDE has a hard time following all the links, often can't, which I find disturbing).  I noticed it's the same way (int32) in 212x64 (but not 128x64).